srcpkgs/gala/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'gala'
+pkgname=gala
+version=3.2.0
+revision=1
+build_style=meson
+hostmakedepends="cmake pkg-config gettext glib-devel"
+makedepends="gnome-desktop-devel vala-devel bamf-devel plank-devel granite-devel gnome-settings-daemon-devel mutter-devel libcanberra-devel"
```

Comment:
Likewise, please use the commit for gala from #20429 as-is. These dependencies have a couple issues:

* cmake is not necessary
* vala-devel is not appropriate

And the formatting is wrong. We wrap lines at 80 characters when possible.

Reply:
What about[ this line](https://github.com/void-linux/void-packages/pull/20429/commits/3cdcfceaf1d7812f99b284bd0fa6fe70759da47b#diff-e43402fceea4b09470acd3f61cf015e4R23)?

This is what @xtraeme  said in above messages:
> this is definitely wrong. shared libraries should be part of the main pkg, not development.

Reply:
@xtraeme What's your idea?

Reply:
If it ends in .so, it goes in the devel package. If it ends in a number, it goes in the main package.

Reply:
well, this is not completely right. The real shared library might not be versioned at all. shared libraries are those detected as "x-sharedlib" by `file -bi`. That means that they are ELF objects with `SONAME` on it.

But in 99% of cases @CameronNemo is right. Usually the .so files are symlinks to the real shared library. Those .so symlinks are only required when linking against it, so that they are only required in devel pkgs.